### PR TITLE
[refactor] odemis.gui.cont.acquisition.py file

### PR DESCRIPTION
**Pull request by nandishjpatel**
[refactor] create a acquisition subpackage in odemis.gui.cont, all the classes earlier in odemis.gui.cont.acquisition.py have been split into individual python files under the odemis.gui.cont.acquisition subpackage.

The refactoring will result in a better file structure.

### CODE REVIEW

File: src/odemis/gui/cont/acquisition/snap_shot.py
Comment (pieleric) [inline src/odemis/gui/cont/acquisition/snap_shot.py:1]: Let's name this file `snapshot.py`. No `_`.

File: src/odemis/gui/cont/acquisition/fine_align.py
Comment (pieleric) [inline src/odemis/gui/cont/acquisition/fine_align.py:1]: This file has nothing to do with acquisition => move it to `gui.cont`

File: src/odemis/gui/cont/acquisition/auto_center.py
Comment (pieleric) [inline src/odemis/gui/cont/acquisition/auto_center.py:1]: This file has nothing to do with acquisition => move it to `gui.cont`

**Review by pieleric — CHANGES_REQUESTED** [review]

File: src/odemis/gui/cont/acquisition/fine_align.py [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+import wx
+
+from odemis import model
+import odemis.gui.model as guimod
+from odemis.acq import align
+from odemis.gui.comp import popup
+from odemis.gui.comp.canvas import CAN_DRAG, CAN_FOCUS
+from odemis.gui.util import call_in_wx_main
+from odemis.gui.util.widgets import ProgressiveFutureConnector
+from odemis.util import units
```
Comment (tepals) [inline src/odemis/gui/cont/acquisition/fine_align.py:43]:
```suggestion
import odemis.gui.model as guimod
from odemis import model
from odemis.acq import align
from odemis.gui.comp import popup
from odemis.gui.comp.canvas import CAN_DRAG, CAN_FOCUS
from odemis.gui.util import call_in_wx_main
from odemis.gui.util.widgets import ProgressiveFutureConnector
from odemis.util import units
```

**Review by tepals — APPROVED** [review]

**Review by pieleric — APPROVED** [review]

**Pull request merged**